README.md

```diff
@@ -37,10 +36,11 @@ to capture detailed changes at the individual row level.
 
 ### Configuration
 
-| name     | description                                                                                                                    | required | default value | example                                                   |
-| -------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- | ------------- | --------------------------------------------------------- |
-| `dsn`    | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database. | true     |               | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true |
-| `tables` | The list of tables to pull data from                                                                                           | true     |               | users,posts,admins                                        |
+| name                                     | description                                                                                                                                             | required | default value | example                                                         |
+| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- | --------------------------------------------------------------- |
+| `dsn`                                    | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database.                          | true     |               | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true       |
```

Comment:
`<user>:<password>` is not shown when the markdown is generated.

Reply:
Oh good catch, the markdown html thing.